src/vs/workbench/contrib/inlineChat/browser/inlineChatDecorations.ts

```diff
@@ -113,6 +106,7 @@ export class InlineChatDecorationsContribution extends Disposable implements IEd
 		const editor = this._editor;
 		const decorationUpdateScheduler = new RunOnceScheduler(() => this._onSelectionOrContentChanged(editor), 100);
 		this._localToDispose.add(decorationUpdateScheduler);
+		this._localToDispose.add(this._debugService.getModel().onDidChangeBreakpoints(() => this._updateCurrentBreakpoints(editor.getModel().uri)));
```

Comment:
I think you also need to schedule a decorator update in onDidChangeBreakpoints

Reply:
That's right, my bad thank you for noticing.